### ISSUE TITLE
Creating an empty folder

### ISSUE THREAD

**Issue by kpcyrd** [mmstick/cargo-deb#124]
Hello!

I'm trying to create an empty folder in /etc/ that is going to hold config files that the user is going to write.

This could be implemented with:
```
assets = [
    # binary
    ["target/release/foo", "usr/bin/", "755"],
    # directory
    ["etc/my/empty/folder/", "755"],
]
```
thanks!

**Comment by kornelski** [comment]
How does that work in Debian? Is it enough to just make the dir in the tarball?

**Comment by kpcyrd** [comment]
Yes. If you want lintian to be happy you'd also need to add an override for `package-contains-empty-directory`:
https://lintian.debian.org/tags/package-contains-empty-directory.html

**Comment by ximon18** [comment]
If you are using a systemd unit you can also use the [`ConfigurationDirectory`](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#RuntimeDirectory=) directive.
